projects/plugins/jetpack/modules/subscriptions/jetpack-user-content-link-redirection.php

```diff
@@ -9,17 +9,38 @@
  * @package automattic/jetpack
  */
 
+use Automattic\Jetpack\Connection\Client;
+
 /**
  * Render a page containing an iframe to track and redirect the user content link in emails.
  */
 function jetpack_user_content_link_redirection() {
 	if ( empty( $_SERVER['QUERY_STRING'] ) ) {
 		return;
 	}
+
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	if ( isset( $_GET['user_content_redirect_domain_checking'] ) ) {
```

Comment:
Is this functioning as a feature flag for now? It's kinda long for something we'd like to hide in obscurity.

Reply:
It was intended to be used for tests only, but the changes are not as extensive as we expected, so it's no big deal to remove it.

Reply:
Ah okay, then we can leave it in.

Reply:
Removed as we changed for an even smaller approach. There won't be any need for a flag here.